.scalafmt.conf

```diff
@@ -1,11 +1,8 @@
-version = 3.0.7
+version = 3.7.2
 
 runner.dialect = scala213
-fileOverride {
```

Comment:
This is no longer necessary since `project.layout = standardConvention` automatically applies this dialect to `scala-3` folders (assuming you have the standard sbt layout which this project does).